README.md

```diff
@@ -800,7 +802,7 @@ const index = pc.index('my-index');
 const results = await index.query({ topK: 10, id: '1' });
 ```
 
-#### Hybrid search with sparseVector
+#### Hybrid search with sparse vectors
```

Comment:
Updating this example while I'm here in this PR -- unrelated to language, but we were using a pretty outdated image search example (really, that's "multi-modal" search) that I didn't find intuitive.